src/controller/python/chip/server/ServerInit.cpp

```diff
@@ -138,6 +147,17 @@ void pychip_server_native_init()
         ChipLogError(DeviceLayer, "Failed to initialize CHIP stack: platform init failed: %s", chip::ErrorStr(err));
     }
 
+#if CHIP_DEVICE_LAYER_TARGET_LINUX
+    err = chip::examples::InitCommissionableDataProvider(gCommissionableDataProvider, LinuxDeviceOptions::GetInstance());
```

Comment:
How are command line options parsed here? How do we ensure no clash with the Python command line options?

Reply:
Using `LinuxDeviceOptions` only works for C++ based example applications. It should not be used here.

I'd recommend this instead:

```
    static chip::DeviceLayer::TestOnlyCommissionableDataProvider TestOnlyCommissionableDataProvider;
    chip::DeviceLayer::SetCommissionableDataProvider(&TestOnlyCommissionableDataProvider);
```

Reply:
@mrjerryjohns thank for the recommendation, I've updated in https://github.com/project-chip/connectedhomeip/pull/22498/commits/5de23efe466e49b7cb67f9139c923b6b71dffcee.